analytic_distribution_filter/models/analytic_account_line.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import Command, api, fields, models
+
+
+class AnalyticAccountLine(models.Model):
+    _name = "analytic.account.line"
```

Comment:
To better represent what it is.
```suggestion
    _name = "analytic.account.relation.line"
    _description = "Analytic Account Relation Line"
```